jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -29,9 +29,9 @@
             'harness_config': os.path.join('firefox_ui_tests', 'qa_jenkins.py'),
             'harness_script': os.path.join('firefox_ui_tests', 'update.py'),
             'treeherder': {
-                'group_name': 'Firefox UI Tests - update',
-                'group_symbol': 'Fu',
-                'job_name': 'Firefox UI Tests - update ({locale}-{update_number})',
+                'group_name': 'Firefox UI Update Tests - {update_channel}',
+                'group_symbol': 'Fu-{update_channel}',
```

Comment:
Hm, no I haven't. But we already have this problem. So I would propose to have another issue where we can discuss that because it may also affect the functional tests group symbol, ie. if we want to use FxF/FxU for our tests. We should get this other issue fixed first before we push this PR to staging for sure. Would that work for you?